device/ingrasys/x86_64-ingrasys_s9130_32x-r0/INGRASYS-S9130-32X/port_config.ini

```diff
@@ -0,0 +1,33 @@
+# name          lanes             alias             port
```

Comment:
Change final column title from `port` to `index`.

Refer to https://github.com/Azure/SONiC/wiki/Porting-Guide.